dbt-metricflow/dbt_metricflow/cli/mf_tutorial_project/profiles.yml

```diff
@@ -3,4 +3,4 @@ mf_tutorial:
   outputs:
     dev:
       type: duckdb
-      path: 'mf_tutorial.duckdb'
+      path: $db_file_path
```

Comment:
wonder if it's worth doing like,
```
> mf_tutorial_project/
     - templated_profiles.yml
     - profiles.yml
```
Where `profiles.yml` is a default valid one like the current one. Then you just read from `templated_profiles.yml` which contains all the formatting strings, and you overwite it into `profiles.yml`? Just wondering if like the user haven't ran the tutorial yet and just looked at this `profiles.yml` would it cause slight confusion.

Reply:
> Just wondering if like the user haven't ran the tutorial yet and just looked at this profiles.yml

I thought it only gets created when the tutorial is run?

Reply:
Oh yea thats right